bin/civi-test-run

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -e
+
+if [ -z $1 ]; then
+  echo "No Buildkit path passed in"
+  exit 2
+fi
+
+if [ -z $2 ]; then 
+  echo "No Build Name passed in"
+  exit 2
+fi
+
+if [ -z $3 ]; then
+  echo "No CiviCRM Version passed in"
+  exit 2
+fi
+
+if [ -z $4 ]; then
+  echo "No JUNIT Dir passed in"
+  exit 2
+fi
+
+if [ -z $5 ]; then
+  echo "No Test type passed in"
+  exit 2
+fi
+
+BLDKIT=$1
+BLDNAME=$2
+CIVIVER=$3
+BLDDIR="$BLDKIT/build/$BLDNAME"
+JUNITDIR=$4
+TESTTYPE=$5
+EXITCODES=
+GUARD=
+
+#################################################
+## Helpers
+function fatal() {
+  echo "$@"
+  exit 1
+}
+
+## RUN PHP Unit tests
+function runPHPUnit() {
```

Comment:
The use of `$1`, etal, is a bit obscure -- adding some sort of docblock/signature would help. e.g.

```bash
## Execute a PHPUnit suite
## 
## Usage: runPHPUnit <test>
## Ex: runPHPUnit PHPUNIT-API
function runPHPUnit() { 
```

Reply:
@totten this is no longer needed i believe